stream_chat/tests/test_channel.py

```diff
@@ -82,6 +82,12 @@ def test_truncate(self, channel):
         response = channel.truncate()
         assert "channel" in response
 
+    def test_truncate_with_options(self, channel):
+        response = channel.truncate(
+            skip_push=True, message={"text": "Truncating channel."}
```

Comment:
you can add `"user_id":random_user["id"]`

Reply:
it should be enough to make it passed

Reply:
✔️